spec/lib/vanagon/platform/windows_spec.rb

```diff
@@ -10,6 +22,7 @@
       :output_dir             => "windows/x64",
       :output_dir_with_target => "windows/thing/x64",
       :target_user            => "Administrator",
+      :projname               => "test-proj",
```

Comment:
Since we're never using this to make sure vanagon sets the project name correctly, you could always just pass a random string into the method to generate the packaging artifacts. That's just a superficial suggestion though, and doesn't really matter